src/components/header/template.njk

```diff
@@ -56,38 +55,29 @@
         {% endif %}
       </a>
     </div>
-
-    {% if params.serviceName or params.navigation or params.signOutHref or params.languageToggle %}
+    {% if params.serviceName or params.navigation | length %}
     <div class="govuk-header__content">
-      {% if params.serviceName %}
-        <a href="{{ params.serviceUrl }}" class="govuk-header__link govuk-header__link--service-name">
-          {{ params.serviceName }}
-        </a>
-      {% endif %}
-
-      {% if params.navigation %}
-        <button type="button" role="button" class="govuk-header__menu-button js-header-toggle"
-                aria-controls="navigation"
-                aria-label="Show or hide Top Level Navigation">Menu
-        </button>
-        <nav>
-          <ul id="navigation"
-              class="govuk-header__navigation {{ params.navigationClasses if params.navigationClasses }}"
-              aria-label="Top Level Navigation">
-            {% for item in params.navigation %}
-              {% if item.href and item.text %}
-                <li class="govuk-header__navigation-item{{ ' govuk-header__navigation-item--active' if item.active }}">
-                  <a class="govuk-header__link"
-                     href="{{ item.href }}"{% for attribute, value in item.attributes %} {{ attribute }}="{{ value }}"{% endfor %}
-                  >
-                  {{ item.text }}
-                  </a>
-                </li>
-              {% endif %}
-            {% endfor %}
-          </ul>
-        </nav>
-      {% endif %}
+    {% if params.serviceName %}
+    <a href="{{ params.serviceUrl }}" class="govuk-header__link govuk-header__link--service-name">
```

Comment:
There's a discussion going on at the moment about if the service name is displayed but serviceUrl isn't supplied. Not a change for this PR necessarily but wanted to mention.

Reply:
Yeah, it's a good conversation to have but I'd see it as separate to this PR.